dsa/validate_test.go

```diff
@@ -11,14 +11,25 @@ import (
 )
 
 func TestValidate(t *testing.T) {
+	validBehalf := "123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|"
+	invalidBehalf := "123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|1"
+	validPaid := "123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|"
+	invalidPaid := "123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|123456789|1"
```

Comment:
If the key difference is their length, consider using something like `strings.Repeat("a", 100)` to make the length more obvious.